packages/utxo-lib/src/bip32.ts

```diff
@@ -6,7 +6,7 @@
 // - `fromBase58` and `toBase58` methods are using additional "network" param in bs58check.encode/decode (Decred support).
 
 import ecc from 'tiny-secp256k1';
-import wif from 'wif';
+import * as wif from 'wif';
```

Comment:
:thought_balloon: Do we prefer to import everything from ES exports?
Why not `import { encode } from 'wif'`
or `import { encode as wifEncode } from 'wif'` to make the usage clearer?

Same with `bip66`..